### ISSUE TITLE
[BUG] Incompatible with Skip Redirect for Firefox

### ISSUE THREAD

**Issue by cachho** [cachho/reparchive-browser-extension#66]
Firefox Addon Link: https://addons.mozilla.org/en-US/firefox/addon/skip-redirect/

A user reported that the extension does not work if this addon is installed.

**Comment by cachho** [comment]
It seems like this extension extracts any URL from the query params and replaces the link with them so `agent.com/item?url=weidian.com?id=123` would be replaced with `weidian.com?id=123`. 

This sounds like it would generally be a problem and make all agent links redirect to the marketplace, no matter what. **Effectively, you can't use agent links.** This is an issue that's bigger than this extension. It seems like you can (and have to) blacklist all agents and marketplaces to not be affected by the "skip redirect" extension. The mozilla addons description calls this blacklist `no-skip-urls-list`.

In other words, I don't see how I could solve this problem, and the extension is causing so many other issues in our field that you should use a blacklist or not use the extension at all.

I'll close this issue because it is not related to this extension, and building a workaround to ensure compatibility is out of scope.

This is all based on the description mozilla addon description and has not been tested.
